clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/GCEServerGroupNameResolver.groovy

```diff
@@ -48,7 +51,7 @@ class GCEServerGroupNameResolver extends AbstractServerGroupNameResolver {
 
   @Override
   List<AbstractServerGroupNameResolver.TakenSlot> getTakenSlots(String clusterName) {
-    def managedInstanceGroups = GCEUtil.queryAllManagedInstanceGroups(project, region, credentials, task, phase)
+    def managedInstanceGroups = GCEUtil.queryAllManagedInstanceGroups(safeRetry, project, region, credentials, task, phase)
```

Comment:
Any particular reason to have `safeRetry` at the beginning of the arg list here, but at the end of the constructor up above? If you're adding a new arg, I'd usually expect it to just go at the end of the list. But wherever you put it, let's make it consistent.